CMakeLists.txt

```diff
@@ -565,7 +565,7 @@ function(fw_add_variant variant_name)
       # Patched binary with pre-baked secondary language
       add_custom_command(
         OUTPUT ${FW_LANG_FINAL}.bin
-        COMMAND ${CMAKE_OBJCOPY} -O binary ${FW_LANG_BASE} ${FW_LANG_FINAL}.bin
+        COMMAND ${CMAKE_COMMAND} -E copy ${FW_LANG_PATCH}.bin ${FW_LANG_FINAL}.bin
         COMMAND "${Python3_EXECUTABLE}" ${CMAKE_SOURCE_DIR}/lang/lang-patchsec.py ${FW_LANG_BASE} ${LANG_BIN}
                 ${FW_LANG_FINAL}.bin
         DEPENDS ${FW_LANG_BASE} ${LANG_BIN}
```

Comment:
should add `${FW_LANG_PATCH}.bin` to DEPENDS so the dependency tree is ordered correctly.

Reply:
Done